.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+<!--
+These comments inside these brackets will not appear in the pull request.
+
+The pull request should be linked to either:
+ - a task (i.e., use 'Closes #TaskID' or 'Resolves #TaskID')
+ - a bug (i.e., use 'Closes #BugID' or 'Fixes #BugID')
+
+For more details, see: https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
+-->
+
+Closes #TBDTaskIDorBugID
+
+TBDAdditionalPRDetails
```

Comment:
I wonder if there's an assignees field for pull requests, then we could put all of our names on it and have that piece automated as well.

Reply:
We discussed this in our weekly scrum, and we may use a different form of communication for PRs